src/aave-v3/SupplyVault.sol

```diff
@@ -98,17 +98,16 @@ contract SupplyVault is ISupplyVault, SupplyVaultBase {
         claimedAmounts = new uint256[](nbRewardTokens);
 
         for (uint256 i; i < nbRewardTokens; ++i) {
-            address rewardToken = rewardTokens[i];
-            UserRewardsData storage userRewardsData = userRewards[rewardToken][_user];
+            UserRewardsData storage userRewardsData = userRewards[rewardTokens[i]][_user];
```

Comment:
wut it's better to query it at each time in the list?

Reply:
Actually I made this change for the sake of code clarity, as I thought assigning reward token was unnecessary bloat. rewardTokens is already in memory, so the gas cost should already be very low. Can revert if it's not desired.

Reply:
Oh ok, I don't feel the previous version too heavy but ok

Reply:
I don't understand this change: as a reader, it's easier to store a variable in my brain (`rewardToken = rewardTokens[i]`) once and use it afterwards rather than having to check the correct variable is read everytime it's used (ie checking that `rewardTokens[i]` is used)

Reply:
Agree, I think we can revert the change